clients/admin-ui/src/features/privacy-experience/PrivacyExperienceTranslationForm.tsx

```diff
@@ -296,6 +296,15 @@ const PrivacyExperienceTranslationForm = ({
           variant="stacked"
         />
       ) : null}
+      {!!formConfig.trigger_link_label?.included && (
```

Comment:
`.included` should always be a boolean. Can we follow the convention in this file and just use:

```suggestion
      {formConfig.trigger_link_label?.included && (
```

Reply:
with the `?` there's a potential for it to be `undefined` instead of `boolean`, which is why I add the coercion. The result is the same though, so I'm not opposed.

Reply:
FWIW, I believe the use of the ternary operators above are an anti-pattern. Using `&&` in those places is a much cleaner approach. Thoughts?

Reply:
mostly I just want the file to be consistent, so feel free to refactor! I'm all for using && instead of ternary :)